modules/apps/ccv/types/events.go

```diff
@@ -0,0 +1,12 @@
+package types
+
+// IBC transfer events
+const (
+	EventTypeTimeout      = "timeout"
+	EventTypePacket       = "ccv_packet"
+	EventTypeChannelClose = "channel_closed"
+
+	AttributeKeyAckSuccess = "success"
+	AttributeKeyAck        = "acknowledgement"
+	AttributeKeyAckError   = "error"
```

Comment:
everything but the `EventTypePacket` look like they could be imported from 04-channel